samples/dnn/openpose.cpp

```diff
@@ -102,16 +104,17 @@ int main(int argc, char **argv)
     int W = result.size[3];
 
     // find out, which model we have
-    if (nparts == 19)
+    if (nparts == 19 || nparts == 57)
     {   // COCO body
         midx   = 0;
         npairs = 17;
         nparts = 18; // skip background
     }
-    else if (nparts == 16)
+    else if (nparts == 16 || nparts == 44)
     {   // MPI body
         midx   = 1;
         npairs = 14;
+        nparts = 16;
     }
```

Comment:
I think it's better to add a flag `dataset` similar to python script and add an extra option 'HAND' to it's choices instead determining model by number of output channels.